files/helpers/const.py

```diff
@@ -461,7 +461,7 @@
+WORDLE_LIST = ('aaron')
```

Comment:
Why keep just this one?

Reply:
It's possible the whole system would break messily if there wasn't at least one thing in the list, and I didn't want to go to the trouble of removing the entire thing.

Reply:
Alright